docs/commands/rhoas_service-registry_role.adoc

```diff
@@ -0,0 +1,78 @@
+ifdef::env-github,env-browser[:context: cmd]
+[id='ref-rhoas-service-registry-role_{context}']
+= rhoas service-registry role
+
+[role="_abstract"]
+[alpha] Service Registry role management
+
+[discrete]
+== Synopsis
+
+
+Service Registry role management
+Set of commands that give users one of following permissions:
+
+- READ_ONLY (read artifacts)
+- DEVELOPER (write access to all resources)
+- ADMIN (Export/Import artifacts, Manage Roles)
+
+Roles can be applied to users and service accounts generated by cli (service account ids
```

Comment:
Suggest: 
Roles can be applied to users and service accounts generated by the CLI (service account IDs).

These commands are only accessible to users with the organization admin role or owners of the Service Registry instance.

This command is available as part of the Development Preview release.